### PR TITLE
pc - use database_url instead of jdbc_database_url

### PR DESCRIPTION
**Pull request by pconrad**
In this PR, we try using DATABASE_URL instead of JDBC_DATABASE_URL for dokku.

### CODE REVIEW

File: src/main/resources/application-production.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-spring.datasource.url=${JDBC_DATABASE_URL}
+spring.datasource.url=${DATABASE_URL}
```
Comment (pconrad) [inline src/main/resources/application-production.properties:1]:
```suggestion
spring.datasource.url=jdbc:${DATABASE_URL}
```

**Review by bnieder — APPROVED** [review]
LGTM

**Review by bnieder — CHANGES_REQUESTED** [review]
If you are going to do this, I wonder whether we might also need to update the documentation about these environment variables?

**Comment by bnieder** [comment]
LGTM

**Comment by pconrad** [comment]
Unnecessary... and would cause a lot of documentation to need to change.

**Pull request closed without merging**